src/main/java/org/entando/entando/aps/servlet/StartupListener.java

```diff
@@ -13,23 +13,48 @@
  */
 package org.entando.entando.aps.servlet;
 
+import com.agiletec.aps.system.SystemConstants;
+import org.entando.entando.aps.system.exception.CSRFProtectionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import javax.servlet.ServletContext;
 import javax.servlet.ServletContextEvent;
+import java.util.Objects;
 
 /**
  * Init the system when the web application is started
+ *
  * @author E.Santoboni
  */
 public class StartupListener extends org.springframework.web.context.ContextLoaderListener {
-	
-	@Override
-	public void contextInitialized(ServletContextEvent event) {
-		ServletContext svCtx = event.getServletContext();
-		String msg = this.getClass().getName()+ ": INIT " + svCtx.getServletContextName();
-		System.out.println(msg);
-		super.contextInitialized(event);
-		msg = this.getClass().getName() + ": INIT DONE "+ svCtx.getServletContextName();
-		System.out.println(msg);
-	}
-	
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StartupListener.class);
+
+    @Override
+    public void contextInitialized(ServletContextEvent event) {
+        ServletContext svCtx = event.getServletContext();
+        String msg = this.getClass().getName() + ": INIT " + svCtx.getServletContextName();
+        System.out.println(msg);
+        super.contextInitialized(event);
+        msg = this.getClass().getName() + ": INIT DONE " + svCtx.getServletContextName();
+        System.out.println(msg);
+
+        boolean isActive = Objects.nonNull(System.getenv(SystemConstants.ENTANDO_CSRF_PROTECTION));
+        String whiteList = System.getenv(SystemConstants.ENTANDO_CSRF_ALLOWED_DOMAINS);
+
+        if (!isActive || !SystemConstants.CSRF_BASIC_PROTECTION.equals(System.getenv(SystemConstants.ENTANDO_CSRF_PROTECTION))) {
+            LOGGER.warn("CSRF protection is not enabled");
+        } else if (SystemConstants.CSRF_BASIC_PROTECTION.equals(System.getenv(SystemConstants.ENTANDO_CSRF_PROTECTION))) {
+            if (whiteList != null && !whiteList.equals("")) {
+                String message = "CSRF protection is enabled Domains --> ".concat(whiteList);
+                LOGGER.info(message);
+            } else {
+                LOGGER.error("CSRF protection is enabled but the domains are initialized. Please initialize the domains");
```

Comment:
```suggestion
                LOGGER.error("CSRF protection is enabled but the domains are not initialized. Please initialize the domains");
```

Reply:
Thanks, for the moment we can to postpone this.